### DISCUSSION TITLE
Feature request.

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#20905] [Q&A]
Kindly introduce a feature on Ubo so that one can prevent loading of any third party social media embed post and embed video player of third party video hosting website on any website. I think this will good for user privacy and also help to increase the loading speed of a website.

**Comment by ghost** [comment]
This is possible by either using dynamic filtering (advanced mode), or filters.

Example filters for YouTube:

```
||youtube.com/embed/$3p,frame,redirect=click2load.html
||youtube-nocookie.com/embed/$3p,frame,redirect=click2load.html
```

As a bonus, these filters allow you to easily allow the frame (frame is added by uBlock Origin and has a button)

**Reply by krystian3w** [reply]
Can fail: https://github.com/uBlockOrigin/uAssets/discussions/19732

https://github.com/uBlockOrigin/uBlock-issues/issues/1938

Someone can test in AdGuard (maybe found workarounds).

**Comment by gorhill** [comment]
https://twitter.com/gorhill/status/1377613392559636486
